src/compute-types/src/dyncfgs.rs

```diff
@@ -199,6 +199,30 @@ pub const ENABLE_COMPUTE_RENDER_FUELED_AS_SPECIFIC_COLLECTION: Config<bool> = Co
     "When enabled, renders `as_specific_collection` using a fueled flat-map operator.",
 );
 
+/// Whether to apply logical backpressure in compute dataflows.
+pub const ENABLE_COMPUTE_LOGICAL_BACKPRESSURE: Config<bool> = Config::new(
+    "enable_compute_logical_backpressure",
+    false,
+    "When enabled, compute dataflows will apply logical backpressure.",
+);
+
+/// Maximal number of capabilities retained by the logical backpressure operator.
+///
+/// The default is sufficiently large that we should rarely hit it in practice.
```

Comment:
Not so sure about this! If we have a "retained metrics collection" that keeps a compaction window of 30 days and is updated every minute (like the storage statistics collections are), we'll end up with 43,200 timestamps at least, but I think maybe twice that with the way our "lag writes by" read policy rounds.

Reply:
Fair point. I'm not sure what a good default value is. Too small and we'll not benefit from the feature, too large and we risk taking forever to hydrate and use more resources in the operator. I'll leave a comment to that effect.

Reply:
I think it should be safe to use a larger "slack" value, like 1 minute or 10 minutes. That would mitigate the taking forever part at least. Resource usage is still a concern though.